addons/mail/static/src/model/store.js

```diff
@@ -19,10 +19,29 @@ export class Store extends Record {
         return this.recordByLocalId.get(localId);
     }
 
+    onBusinessError(err) {
+        this._.FC_QUEUE.clear();
+        this._.FS_QUEUE.clear();
+        this._.FA_QUEUE.clear();
+        this._.FD_QUEUE.clear();
+        this._.FU_QUEUE.clear();
+        this._.RO_QUEUE.clear();
+        this._.RD_QUEUE.clear();
+        this._.RHD_QUEUE.clear();
+        this._.UPDATE = 0;
```

Comment:
Do we actually need to cancel everything in this case?
Can't we just cancel only the current failing operation instead, and at least give a chance to the others?

Reply:
So you would suggest to handle errors only at very end of update cycle?